react/features/prejoin/actions.js

```diff
@@ -73,11 +70,7 @@ const STATUS_REQ_CAP = 45;
  * @param {number} count - The number of retried calls. When it hits STATUS_REQ_CAP it should no longer make requests.
  * @returns {Function}
  */
-function pollForStatus(
```

Comment:
Please don't make whole-file style changes like these. It adds churn, makes the review harder and future git archaeology adventures.